.github/actions/notarize-macos/action.yml

```diff
@@ -32,32 +34,26 @@ runs:
       shell: bash
       run: find build -name "DB Browser for SQL*.app" -exec /opt/homebrew/opt/db4subqt@5/bin/macdeployqt {} -sign-for-notarization="${{ inputs.DEV_ID }}" \;
 
-    - name: Add the extension to the app bundle
+    - name: Add the 'nalgeon/sqlean' extension to the app bundle
       shell: bash
       run: |
+        gh auth login --with-token <<< "${{ inputs.GH_TOKEN }}"
+        gh release download --pattern "sqlean-macos-arm64.zip" --repo "nalgeon/sqlean"
+        unzip sqlean-macos-arm64.zip
         for TARGET in $(find build -name "DB Browser for SQL*.app" | sed -e 's/ /_/g'); do
           TARGET=$(echo $TARGET | sed -e 's/_/ /g')
           mkdir "$TARGET/Contents/Extensions"
-          clang -I /opt/homebrew/opt/db4subsqlitefts@5/include -L /opt/homebrew/opt/db4subsqlitefts@5/lib -fno-common -dynamiclib src/extensions/extension-formats.c -o "$TARGET/Contents/Extensions/formats.dylib"
-          clang -I /opt/homebrew/opt/db4subsqlitefts@5/include -L /opt/homebrew/opt/db4subsqlitefts@5/lib -fno-common -dynamiclib src/extensions/extension-functions.c -o "$TARGET/Contents/Extensions/math.dylib"
 
+          clang -I /opt/homebrew/opt/db4subsqlitefts@5/include -L /opt/homebrew/opt/db4subsqlitefts@5/lib -fno-common -dynamiclib src/extensions/extension-formats.c -o "$TARGET/Contents/Extensions/formats.dylib"
```

Comment:
Some of these lines have `.dylib` on the end, while others have it twice.

From rough memory, the doubled version of `.dylib` was to work around some problem when loading the extension, though I don't remember the details. :wink:

That being said, I'm wondering if they should be consistent?  aka should they all be `.dylib` or `.dylib.dylib` rather than a mix of the two?